packages/contact-center/station-login/src/station-login/station-login.presentational.tsx

```diff
@@ -1,137 +1,60 @@
-      color: '#0052bf'
-    },
-
-    btn: {
-      padding: '10px 20px',
-      backgroundColor: '#0052bf',
-      color: 'white',
-      border: 'none',
-      borderRadius: '4px',
-      cursor: 'pointer',
-      transition: 'background-color 0.3s',
-      marginRight: '8px' 
-    },
-
-    select: {
-      width: '100%',
-      padding: '8px',
-      marginTop: '8px',
-      marginBottom: '12px',
-      border: '1px solid #ccc',
-      borderRadius: '4px'
-    },
-
-    input: {
-      width: '97%',
-      padding: '8px',
-      marginTop: '8px',
-      marginBottom: '12px',
-      border: '1px solid #ccc',
-      borderRadius: '4px'
-    }
-  }
+  // The below is a dummy representation of the new CC Widget UI, the below required props can be extracted from the props object and passed to the uuip-cc-user-profile Web Component.
 
   return (
-    <><h1 data-testid="station-login-heading">{name}</h1>
-      <div style={styles.box}>
-      <section style={styles.sectionBox}>
-        <fieldset style={styles.fieldset}>
-        <legend style={styles.legendBox}>Agent</legend>
-          <div style={{ display: 'flex', flexDirection: 'column', flexGrow: 1 }}>
-            <div style={{ display: 'flex', gap: '1rem' }}>
-              <fieldset style={{border: '1px solid #ccc', borderRadius: '5px', padding: '10px', marginBottom: '20px', flex: 0.69 }}>
-                <legend style={styles.legendBox}>Select Team</legend>
-                <select id="teamsDropdown" style={styles.select}>Teams</select>
-              </fieldset>
-              <fieldset style={styles.fieldset}>
-                <legend style={styles.legendBox}>Agent Login</legend>
-                <select name="LoginOption" id="LoginOption" style={styles.select} onChange={selectLoginOption}>
-                  <option value="" selected hidden>Choose Agent Login Option...</option>
-                </select>
-                <input style={styles.input} id="dialNumber" name="dialNumber" placeholder="Extension/Dial Number" type="text" onInput={updateDN} />
-                <button id="AgentLogin" style={styles.btn} onClick={login}>Login</button>
-                <button id="logoutAgent" style={styles.btn} onClick={logout}>Logout</button>
-              </fieldset>
-            </div>
-          </div>
-        </fieldset> 
-      </section>
-    </div></>
+    <div>
+      <style>
+        {`
+          .user-profile-section .profile-settings {
+            cursor: pointer;
+          }
+
+          .user-profile-section .profile-settings:hover {
+            background-color: var(--button-white-hover-bg-color);
+            border-radius: 8px;
+          }
+
+          .profile-settings {
+            display: flex;
+            flex-direction: column;
+            margin: 16px;
+          }
+        `}
+      </style>
+      <h3>This is the new CC Widget UI</h3>
+      <div className="profile-settings">
+        <div className="user-profile-wrapper">
+          <uuip-wc-user-profile
```

Comment:
**The below is a dummy representation of the new CC Widget UI, the below required props can be extracted from the props object and passed to the uuip-cc-user-profile Web Component.**